src/Exception/BadRequestException.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: kasper
+ * Date: 10/01/2018
+ * Time: 14.38
```

Comment:
Isn't these kinda useless? Doesn't follow the coding standard either.

Reply:
Totally agree. I missed that. I blame it on PhpStorm + comments folding.